src/views/ProfileView/ProfileView.js

```diff
@@ -1,16 +1,26 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './ProfileView.css';
 import UserDetails from '../../components/UserDetails/UserDetails';
 import UserInteractions from '../../components/UserInteractions/UserInteractions';
-// import Feed from '../../components/Feed/Feed';
+import Feed from '../../components/Feed/Feed';
+import { user } from '../../Data/mockData';
 
 const ProfileView = () => {
+  const [currentUser, setCurrentUser] = useState({})
```

Comment:
Related to my other question - is this where we would try to pull in the user.posts? Idk if we have that data structure set yet of course.

Reply:
Right now I have all the posts from the mock data being pulled in to the Feed component but I think it would make sense to move user.posts to this component and pass them as props to Feed. And then we can just fetch all posts in the Feed component.

Reply:
this seems related to what we'd been discussing earlier - looks like we'll be using the boolean value of `profileIsVisible` in state to conditionally render whether all or just the individual user's posts are displayed in the feed, based on which page we're currently viewing. Cool to see the solution you worked out for this approach, Christina!